applications/utilities/utils.py

```
@@ -57,3 +61,108 @@ def is_robot_connected(ip: str) -> bool:
+                delta_pose[1] = aruco_pose[1] + offset['y']     # Add gripper offsets
+                delta_pose[2] = aruco_pose[2] + offset['z']     # Add gripper offsets
+                delta_pose[3] = offset['rx']
+                delta_pose[4] = offset['ry']
+                delta_pose[5] = offset['rz']
+                return delta_pose
+            else:
+                print(f"Position '{target_position}' not found in offsets")
+        else:
+            # Process all positions
+            delta_poses = []
+            for position_name, offset in offsets.items():
```

Ruff (0.12.2)

Loop control variable `position_name` not used within loop body

Rename unused `position_name` to `_position_name`

(B007)